charmhelpers/contrib/openstack/templates/section-keystone-authtoken-mitaka

```diff
@@ -22,4 +22,6 @@ signing_dir = {{ signing_dir }}
 {% if use_memcache == true %}
 memcached_servers = {{ memcache_url }}
 {% endif -%}
+service_token_roles = Admin
+service_token_roles_required = True
```

Comment:
It appears that these were introduced in the [Ocata release for the keystonemiddleware](https://docs.openstack.org/releasenotes/keystonemiddleware/ocata.html) which suggests that these may need to be included in the section-keystone-authtoken template rather than this one?

Additionally, the `Admin` role is a configurable role in the keystone charm, and so hard-coding it to Admin may not work in all cases.